### DISCUSSION TITLE
4/9(일) SceneDelegate

### DISCUSSION THREAD

**Discussion by jjunhaa0211** [Info-iOS/iOStudy#64] [General]
swift의 SceneDelegate가 어떤 역활을 하는지 알아보면 분명 개발하면서 많은 도움이 될 것 같습니다!! 이건 이랬지 하고 쓰는 것과 그냥 view 띄워주는 거네~ 이거는 확실히 차이가 있다고 생각합니다. 이제 정처기도 야무지게 땄으니까 이거 알아보고 요번주 투두까지 하는 것을 추천드려요.😈

**Comment by Eunho0922** [comment]
넵
